.travis.yml

```diff
@@ -16,6 +16,7 @@ addons:
 env:
   global:
     - COMPILER_NAME=clang COMPILER=clang++ CXX=clang++ CC=clang
+    - LLVM_SHORT_VERSION=9
```

Comment:
Perhaps we could read this from `bin/versions` to eliminate redundancy here?

https://stackoverflow.com/questions/59909029/travis-yml-file-should-read-from-another-file-a-value-and-use-it

Reply:
Done.